motoko/ic-pos/README.md

```diff
@@ -148,42 +150,37 @@ The icpos canister manages the store configuration and sends notifications when
 The `--argument '(0)'` argument is used to initialize the canister with `startBlock` set to 0. This is used to tell the canister to start monitoring the ledger from block 0. When deploying to the IC mainnet, this should be set to the current block height to prevent the canister from processing old transactions.
 
 ```bash
-dfx deploy --network local icpos --argument '(0)'
+dfx deploy icpos --argument '(0)'
 ```
 
 ### Step 6: Configure the icpos canister:
 
 ic-pos uses [Courier](https://courier.com/) to send email and SMS notifications. If you want to enable notifications, you need to sign up for a Courier account and and create and API key. Then issue the following command:
 
 ```bash
-dfx canister --network local call icpos setCourierApiKey "pk_prod_..."
+dfx canister call icpos setCourierApiKey "pk_prod_..."
 ```
 
 ### Step 7: Build and run the frontend:
 
 Run npm to install dependencies and start the frontend. The frontend will be available at http://localhost:5173.
 
 ```bash
-npm install
-npm run dev
+pnpm install
+dfx deploy icpos_frontend
 ```
-
-Why don't we deploy the frontend as a local canister? Vite uses lazy loading of modules. This does not work when deploying to a local canister. When deploying to the IC mainnet, this is not an issue. Also, running using `npm run dev` allows for hot reloading of the frontend code when making changes.
-
 ### Step 8: Make a transfer!
 
 Now that everything is up and running, you can make a transfer to your newly created store.
 
-Transfers made from the owner principal will not trigger notifications in the UI since they are regarded as `mint` transactions. To test notifications, you need to make a transfer from another principal.
-
 The easiest way to do this is to create two stores using two different Internet Identity accounts, using two different web browsers. Then, transfer some tokens from one store to the other.
 
 #### 8.1: Create the first store and supply it with some tokens:
 
 Log in to the frontend using the Internet Identity. Configure the store and navigate to the `Receive` page. Click on the principal pill to copy the address to your clipboard. Then, using the `dfx` command, mint some tokens from your owner principal to the store principal.
```

Comment:
i don't see a receive page on the frontend, same for `8.2`

Reply:
I'll make a new video recording as the old one was out of date.